locations/spiders/molton_brown.py

```
@@ -0,0 +1,36 @@
+    }
+
+
+    def post_process_item(self, item: Feature, response: TextResponse, feature: dict) -> Iterable[Feature]:
+        if feature["tag_ids"] in [["25269"],["25270"],["24837"]]:
+            item["branch"] = item.pop("name").replace("Molton Brown","").strip()
+            if feature["store_business_hours"]:
+                print(feature["store_business_hours"])
+                oh = OpeningHours()
+                for times in feature["store_business_hours"]:
+                    if times["open_24hrs"] is not False:
+                       oh.add_range(DAYS_FULL[int(times["week_day"]) - 1], "00:01", "23:59")
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)